content/applications/sales/point_of_sale/employee_login.rst

```diff
@@ -1,81 +1,74 @@
-====================
-Log in with employee
-====================
+==========================
+Multi employees management
+==========================
 
-With Odoo *Point of Sale*, you can manage multiple cashiers. This
-feature allows you to keep track of who is working, when and how much
-each cashier made for that session.
+Odoo Point of Sale allows you to manage access to a defined POS through the **Multi Employees per
+Session** feature. Once enabled, you can select which individuals can log into the POS and keep
+track of the employees involved in each order.
```

Comment:
I think users works great
I've taken part of the suggestion (the first part + the word "users", but no the "then tatatat" since it would imply that I'm giving "steps". Here, I just describe what happens when the feature is activated (for clarity, I'll chance "once enabled", though